src/components/missions.jsx

```diff
@@ -1,57 +1,76 @@
-const Mission = () => (
-  <div className="container mission-holder">
-    <table className="table table-bordered">
-      <thead>
-        <tr>
-          <th>Mission</th>
-          <th>Description</th>
-          <th colSpan={2}>Status</th>
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Same here. It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉